l10n_ve_fiscal_book/invoice.py

```diff
@@ -182,11 +182,11 @@ def _get_doc(self, cr, uid, ids, name, args, context=None):
             ret.update({i: ''})
         if invs:
             for inv in invs:
-                if ((inv.type == "in_invoice" or inv.type == "out_invoice")
-                        and inv.parent_id):
+                if (inv.type == "in_invoice" or
```

Comment:
if performing this changes,

Better do:
if inv.type in ('in_invoice', 'out_invoice') and inv.parent_id: